app/src/main/java/com/epmedu/animeal/tabs/TextInputField.kt

```diff
@@ -0,0 +1,110 @@
+package com.epmedu.animeal.tabs
+import android.content.res.Configuration.UI_MODE_NIGHT_NO
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.material.TextField
+import androidx.compose.material.TextFieldDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.epmedu.animeal.base.theme.AnimealTheme
+import com.epmedu.animeal.base.theme.CursorColor
+import com.epmedu.animeal.base.theme.DarkerGrey
+import com.epmedu.animeal.base.theme.LynxWhite
+import com.epmedu.animeal.base.theme.NavigationItemColor
+
+@Composable
+fun TextInputField(
+    modifier: Modifier,
+    title: String,
+    placeholder: String,
+    onValueChange: (String) -> Unit,
+    value: String
+) {
+
+    Column(
+        modifier = modifier,
+    ) {
+
+        Text(
+            text = title,
+            modifier = Modifier.padding(bottom = 2.dp),
+            style = TextStyle(
+                color = NavigationItemColor,
+                fontSize = 14.sp
+            )
+        )
+
+        TextField(
+            modifier = Modifier.fillMaxWidth().background(
+                color = LynxWhite,
+                shape = RoundedCornerShape(12.dp),
+            ),
+            value = value,
+            onValueChange = {
+                onValueChange(it)
+            },
+            textStyle = TextStyle(color = Color.Black, fontSize = 16.sp),
+            colors = TextFieldDefaults.textFieldColors(
+                textColor = NavigationItemColor,
+                backgroundColor = Color.Transparent,
+                cursorColor = CursorColor,
+                focusedIndicatorColor = Color.Transparent,
+                unfocusedIndicatorColor = Color.Transparent
+            ),
+            placeholder = {
+                Text(
+                    text = placeholder,
+                    fontSize = 16.sp,
+                    color = DarkerGrey
+                )
+            },
+            singleLine = true
+        )
+    }
+}
+
+@Preview(
+    showBackground = true,
+    name = "TextInputField",
+    showSystemUi = true,
+    uiMode = UI_MODE_NIGHT_NO,
+)
+@Composable
+fun TestPreviewInput() {
```

Comment:
made private and renamed to TextInputFieldPreview

Reply:
If I do private 
UnusedPrivateMember:
    active: true was tru so that detekt failed. I did not change detekt.yml file so that did not  made private

Reply:
Done